storybook/src/components/Spotlight/Spotlight.stories.tsx

```diff
@@ -29,6 +36,38 @@ type Story = StoryObj<typeof meta>
 
 export const Default: Story = {}
 
+export const StickOut: Story = {
+  render: ({ as, color }) => (
+    <Grid>
+      <Grid.Cell
+        span="all"
+        rowSpan={{ narrow: 2, medium: 2, wide: 1 }}
+        rowStart={2}
+        style={{
+          marginBlock: 'calc(var(--ams-space-grid-md) * -1)',
+          marginInline: 'calc(var(--ams-space-grid-lg) * -1)',
+        }}
+      >
+        <Spotlight as={as} color={color} style={{ height: '100%' }} />
```

Comment:
We can remove the inline style if we allow the grid cell classes and props on any element – that would make the Spotlight a direct child of the Grid, thus a grid item, stretching by default.

Reply:
We’ve merged this with `coverGap` into `has="spotlight"`.